src/triggered/UpdateSubstation.ts

```diff
@@ -0,0 +1,83 @@
+import { css, html, LitElement, query, TemplateResult } from 'lit-element';
+import { get } from 'lit-translate';
+
+import { newWizardEvent } from '../foundation.js';
+import { Diff, mergeWizard } from '../wizards.js';
+
+function lNSelector(lNode: Element): string {
+  const [iedName, ldInst, prefix, lnClass, lnInst] = [
+    'iedName',
+    'ldInst',
+    'prefix',
+    'lnClass',
+    'lnInst',
+  ].map(name => lNode.getAttribute(name));
+
+  // FIXME: give the correct LN selector string!
+  return lNode.getAttribute('ldInst')
+    ? `:root > IED[name="${lNode.getAttribute('iedName')}"] > AccessPoint > LN`
+    : `:root > IED[name="${lNode.getAttribute(
+        'iedName'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195000)
```suggestion
        "iedName"
```